packages/cli/commands/validate.js

```diff
@@ -0,0 +1,19 @@
+const {
+  addConfigOptions,
+  addAccountOptions,
+  addOverwriteOptions,
+} = require('../lib/commonOpts');
+const validateTheme = require('./validation/validateTheme');
```

Comment:
Might be worth renaming `validation` directory to `validate` to match the pattern used for the other commands where the directory matches the top-level command.

Reply:
Ah I didn't notice that pattern, updated in ee4a6dd 👍